app/objects/c_ability.py

```diff
@@ -18,89 +17,55 @@ class AbilitySchema(ma.Schema):
     technique_id = ma.fields.String(missing=None)
     name = ma.fields.String(missing=None)
     description = ma.fields.String(missing=None)
-    cleanup = ma.fields.List(ma.fields.String(), missing=None)
-    executor = ma.fields.String(missing=None)
-    platform = ma.fields.String(missing=None)
-    payloads = ma.fields.List(ma.fields.String(), missing=None)
-    uploads = ma.fields.List(ma.fields.String(), missing=None)
-    parsers = ma.fields.List(ma.fields.Nested(ParserSchema), missing=None)
-    requirements = ma.fields.List(ma.fields.Nested(RequirementSchema), missing=None)
+    executors = ma.fields.List(ma.fields.Nested(ExecutorSchema()))
+    requirements = ma.fields.List(ma.fields.Nested(RequirementSchema()), missing=None)
```

Comment:
FWIW you don't need to instantiate the schema inside the `Nested(...)` field, you can just pass in the class.

Reply:
Hmm, like if you do `ma.fields.List(ma.fields.Nested(ExecutorSchema))` ? I'd expect that error if you didn't have the `Nested` part.

Reply:
Oh yeah, I misunderstood.